### PR TITLE
adapt dashboard project list item style

### PR DESCRIPTION
**Pull request by vellip**

### CODE REVIEW

File: meinberlin/assets/scss/components/_list_item.scss [hunk 1]
```diff
@@ -17,6 +17,21 @@
...
+.list-item__image {
+    float: left;
+    margin-right: $padding;
+}
+
+.list-item__body {
+    overflow: hidden;
+    padding: 0.7em 0;
+}
+
```
Comment (2e2a) [inline meinberlin/assets/scss/components/_list_item.scss]: Should we then also use this new classes in the other `list-item` occurrences?

File: meinberlin/apps/dashboard2/templates/meinberlin_dashboard2/includes/project_list_item.html [hunk 2]
```diff
@@ -1,36 +1,42 @@
...
-        <span class="list-item__action">
-            <a href="{% extproject_url project 'project-detail' %}"
-               target="{{ project|is_external|yesno:"_blank,_self" }}"
-               class="btn btn--light btn--small">
-                <i class="fa fa-eye" aria-hidden="true"></i>
-                {% if project.is_draft %}
-                {% trans 'Preview' %}
-                {% else %}
-                {% trans 'View' %}
+<li class="list-item{% if project.image %} list-item--has-image{% endif %}">
```
Comment (2e2a) [inline meinberlin/apps/dashboard2/templates/meinberlin_dashboard2/includes/project_list_item.html]: Should also consider `tile_image`.

Comment (vellip) [reply meinberlin/apps/dashboard2/templates/meinberlin_dashboard2/includes/project_list_item.html]: True. I made a template tag for that now.

File: meinberlin/apps/dashboard2/templates/meinberlin_dashboard2/includes/project_list_item.html [hunk 3]
```diff
@@ -1,36 +1,42 @@
...
-               class="btn btn--light btn--small">
-                <i class="fa fa-eye" aria-hidden="true"></i>
-                {% if project.is_draft %}
-                {% trans 'Preview' %}
-                {% else %}
-                {% trans 'View' %}
+{% load i18n project_tags extproject_tags dashboard_tags thumbnail contrib_tags %}
+{% project_tile_image project as project_image %}
+<li class="list-item{% if project_image %} list-item--has-image{% endif %}">
+    {% if project_image %}
```
Comment (2e2a) [inline meinberlin/apps/dashboard2/templates/meinberlin_dashboard2/includes/project_list_item.html]: Do we need a single color box if no image is set, to have the same alignment of the other elements?

Comment (vellip) [reply meinberlin/apps/dashboard2/templates/meinberlin_dashboard2/includes/project_list_item.html]: There we go!

**Comment by 2e2a** [comment]
lgtm

**Pull request merged**